net.agalwood.Motrix.yml

```diff
@@ -0,0 +1,160 @@
+app-id: net.agalwood.Motrix
+branch: stable
```

Comment:
```suggestion
```

Default branch is already `stable`, this is unecessary

Reply:
Fixed